calpool-fe/src/components/Button/Button.js

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+import "./Button.css";
+import { CircularProgress } from "@mui/material";
+
+const Button = ({
+  children,
+  width,
+  height,
+  px,
+  py,
+  bgColor,
+  color,
+  className,
+  onClick,
+  loading,
+  loadColor='white'
+}) => {
+  const style = {
+    width: width || "auto", // Set a default width or pass a specific prop
+    height: height || "auto", // Set a default height or pass a specific prop
+    padding: `${py} ${px}`,
+    color: color,
+    backgroundColor: bgColor,
+  };
+
+  return (
+    <button
```

Comment:
Should you be using the MUI button instead of the raw button especially if there's MUI components here like the CircularProgress bar. Might help with style consistency in the future